### PR TITLE
[terra-functional-testing] Initialize Package

### PR DESCRIPTION
**Pull request by StephenEsser**
### Summary

This PR initializes the terra-functional-testing package. This PR should be used to facilitate any discussions for the package name and initial setup. 

A follow-up PR will be created to initialize the eslint configurations. I'd like to keep these PRs small and specific and hopefully reference the newly scoped @cerner/eslint-config-terra package if those changes make it into master in the near future.

@cerner/terra

### CODE REVIEW

File: packages/terra-functional-testing/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@cerner/terra-functional-testing",
+  "version": "0.0.0",
+  "description": "A functional testing utility for applications built using terra.",
```
Comment (yuderekyu) [inline packages/terra-functional-testing/package.json]:
```suggestion
  "description": "A functional testing utility for applications built using Terra.",
```

**Review by yuderekyu — APPROVED** [review]

File: packages/terra-functional-testing/.npmignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+*.log
+target
+reports
+src
```
Comment (emilyrohrbough) [inline packages/terra-functional-testing/.npmignore:4]: will probably need to add a few things here like config files, error screenshots, tests, etc. so they aren't released with the component

File: packages/terra-functional-testing/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- [Documentation](https://github.com/cerner/terra-toolkit/tree/master/packages/terra-functional-testing/README.md)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install from [npmjs](https://www.npmjs.com): `npm install --save-dev @cerner/terra-functional-testing`
+
+## LICENSE
+
+Copyright 2018-2020 Cerner Innovation, Inc.
```
Comment (emilyrohrbough) [inline packages/terra-functional-testing/README.md]: Are you pulling the original terra-toolkit years for this?

**Review by emilyrohrbough — APPROVED** [review]
Love that this is starting small

File: packages/terra-functional-testing/.npmignore (hunk 2 above)
Comment (StephenEsser) [reply packages/terra-functional-testing/.npmignore:4]: My plan is to add those as they become necessary. Tests can probably be added now

File: packages/terra-functional-testing/README.md (hunk 3 above)
Comment (StephenEsser) [reply packages/terra-functional-testing/README.md]: I copy pasted this from another license, it looks like terra-toolkit started in 2017. Should new packages start in 2020?

Comment (mjhenkes) [reply packages/terra-functional-testing/README.md]: Yeah, start new packages in the year they were created.

File: packages/terra-functional-testing/NOTICE [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Copyright 2018 - 2020 Cerner Innovation, Inc.
```
Comment (mjhenkes) [inline packages/terra-functional-testing/NOTICE]: update me

File: packages/terra-functional-testing/LICENSE [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2018 - 2020 Cerner Innovation, Inc.
```
Comment (mjhenkes) [inline packages/terra-functional-testing/LICENSE]: update me

**Review by emilyrohrbough — APPROVED** [review]

File: packages/terra-functional-testing/package.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@cerner/terra-functional-testing",
```
Comment (mjhenkes) [inline packages/terra-functional-testing/package.json:3]: make this package private until we are ready to release it.

**Review by mjhenkes — APPROVED** [review]

File: packages/terra-functional-testing/package.json (hunk 6 above)
Comment (StephenEsser) [reply packages/terra-functional-testing/package.json:3]: Marked as private: https://github.com/cerner/terra-toolkit/pull/403/commits/6b3fc2df5f64d37f8d470d4df000a1718ad93051

File: packages/terra-functional-testing/LICENSE (hunk 5 above)
Comment (StephenEsser) [reply packages/terra-functional-testing/LICENSE]: Updated year: https://github.com/cerner/terra-toolkit/pull/403/commits/edb681c16b7f21cf4e0ae4c673857b272ab8cee8

File: packages/terra-functional-testing/NOTICE (hunk 4 above)
Comment (StephenEsser) [reply packages/terra-functional-testing/NOTICE]: Updated year: https://github.com/cerner/terra-toolkit/pull/403/commits/edb681c16b7f21cf4e0ae4c673857b272ab8cee8

File: packages/terra-functional-testing/README.md (hunk 3 above)
Comment (StephenEsser) [reply packages/terra-functional-testing/README.md]: License years have been updated: https://github.com/cerner/terra-toolkit/pull/403/commits/805b033f99c127f51c3555129e68c07d72289f27

File: packages/terra-functional-testing/package.json (hunk 1 above)
Comment (StephenEsser) [reply packages/terra-functional-testing/package.json]: Updated: https://github.com/cerner/terra-toolkit/pull/403/commits/76aa92038ac3bc922ff487b62a90c8a2cf7d7739

**Pull request merged**